content/faq/editing/advanced-editing/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: How can I do more advanced editing in OpenStreetMap?
+description: "Tutorial for editing OpenStreetMap with more advanced tools like ID"
+
+taxonomies:
+  faq: ["Editing"]
+
+extra:
+  order: 40
+---
+
+[OpenStreetMap.org](https://www.openstreetmap.org) (OSM) is a database with map data for the entire world. Similar to Wikipedia the data can be used free of charge and is crowd-sourced by millions of volunteers.
+**Organic Maps** uses the OpenStreetMap.org data to generate beautiful and detailed maps for you. 
```

Comment:
Maybe add a paragraph how/when map data is updated here or below.

Reply:
No, I think that rather belongs into a different faq